### PR TITLE
Add gettext calls to the locations pages and the user pages.

### PR DESCRIPTION
**Pull request by bkearney**
These pages provide examples for using single translations:

_("Here is a simple")

single substitutions:

_("I like %s") % "pie"

and complex ones:

_("%{character"} should %{action}") % {:character => "Jar Jar Binks", :action => "be erased"}

### CODE REVIEW

File: app/views/taxonomies/_form.html.erb [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
-        <li><a href="#domains" data-toggle="tab">Domains</a></li>
+        <li><a href="#subnets" data-toggle="tab"><%= _("Subnets") %></a></li>
+        <li><a href="#compute_resources" data-toggle="tab"><%= _("Compute Resources") %></a></li>
+        <li><a href="#media" data-toggle="tab"><%= _("Media") %></a></li>
+        <li><a href="#template" data-toggle="tab"><%= _("Templates") %></a></li>
+        <li><a href="#domains" data-toggle="tab"><%= _("Domains") %></a></li>
       <% end -%>
-      <li><a href="#environments" data-toggle="tab">Environments</a></li>
-      <li><a href="#hostgroups" data-toggle="tab">Host Groups</a></li>
+      <li><a href="#environments" data-toggle="tab"><%= _("Envrionments") %></a></li>
```
Comment (skottler) [inline app/views/taxonomies/_form.html.erb]: Typo in `Envrionments`.

File: locale/app.pot [hunk 2]
```diff
@@ -17,14 +17,278 @@ msgstr ""
...
+msgid "Edit Properties"
+msgstr ""
+
+msgid "Edit User"
+msgstr ""
+
+msgid "Environment"
+msgstr ""
+
+msgid "Envrionments"
```
Comment (skottler) [inline locale/app.pot]: Same type as before.

**Comment by skottler** [comment]
Looks good other than the typo.

**Comment by bkearney** [comment]
Fixed the item found by sam.

**Comment by ohadlevy** [comment]
rebased, clean whitespaces and merged - thanks

**Pull request closed without merging**